telethon/client/chats.py

```diff
@@ -611,5 +611,69 @@ def action(
 
         return _ChatAction(
             self, entity, action, delay=delay, auto_cancel=auto_cancel)
+    
+    async def edit_admin(self, entity, user, change_info=None, post_messages=None, edit_messages=None,
+                         delete_messages=None, ban_users=None, invite_users=None, pin_messages=None, add_admins=None, is_admin=None):
```

Comment:
Since https://github.com/LonamiWebs/Telethon/commit/cd4b915522a7b3d1bd9f392228812a531b58ff7e, we now use type hints, at least in methods of the client, so please adjust as appropriated:

```python
    async def edit_admin(
            self: 'TelegramClient',
            entity: 'hints.EntityLike',
            user: ...,
            *,
            ...) -> ...:
```